File: Simple.cpp
#include<iostream>
#include<string>
using namespace std;
class animal{
    public:
    void display()
    {
        cout<<"I was an animal"<<endl;
    }
};
class cat : public animal{
    public:
    void type(){
        cout<<"I was a cat!!Meow"<<endl;
    }
    //void display(){
      // cout<<"I was an animal and cat"<<endl;
   //}

};
int main()
{

    cat C1;
    C1.display();
    C1.type();
}

File: Simple.java
class Animal{
    void display()
    {
        System.out.println("I was an animal");
    }
}
class Cat extends Animal{
    void type()
    {
        System.out.println("I was a cat ");
    }
}
public class Simple {
    public static void main(String[] args) {
        Cat obj=new Cat();
        obj.display();
        obj.type();
    }
}
